xchange-bitmex/src/test/java/info/bitrich/xchangestream/bitmex/BitmexDeadManSwitchTest.java

```diff
@@ -0,0 +1,95 @@
+package info.bitrich.xchangestream.bitmex;
+
+import info.bitrich.xchangestream.bitmex.dto.BitmexLimitOrder;
+import info.bitrich.xchangestream.core.StreamingExchange;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.knowm.xchange.ExchangeFactory;
+import org.knowm.xchange.ExchangeSpecification;
+import org.knowm.xchange.bitmex.dto.marketdata.BitmexPrivateOrder;
+import org.knowm.xchange.bitmex.dto.trade.BitmexSide;
+import org.knowm.xchange.bitmex.service.BitmexMarketDataService;
+import org.knowm.xchange.bitmex.service.BitmexTradeService;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.utils.CertHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.knowm.xchange.bitmex.BitmexPrompt.PERPETUAL;
+
+/**
+ * @author Nikita Belenkiy on 18/05/2018.
+ */
+public class BitmexDeadManSwitchTest {
+    private static final Logger logger = LoggerFactory.getLogger(BitmexDeadManSwitchTest.class);
+
+    @Test
+    @Ignore
+    public void testDeadmanSwitch() throws Exception {
+        CertHelper.trustAllCerts();
+        BitmexStreamingExchange exchange =
+                (BitmexStreamingExchange) ExchangeFactory.INSTANCE.createExchange(BitmexStreamingExchange.class);
+        ExchangeSpecification defaultExchangeSpecification = exchange.getDefaultExchangeSpecification();
+
+        defaultExchangeSpecification.setExchangeSpecificParametersItem("Use_Sandbox", true);
+
+        defaultExchangeSpecification.setApiKey("QW8Ao_gx38e-8KFvDkFn-Ym4");
```

Comment:
Should these have been committed? (!)

Reply:
This key is carry forward from previous commits. It is an invalid key. I used my own key when I test it. I can change it to "YOUR_API_KEY".

Reply:
No need. I just wanted to make sure no-one had accidentally committed a real key.